packages/.npmrc

```diff
@@ -1 +1,3 @@
 //registry.npmjs.org/:_authToken=${NPM_TOKEN}
+@synthql:registry=https://registry.npmjs.org
```

Comment:
Why did you do these changes? Why are they neeed?

Reply:
Got this from the GitHub documentation for publishing to NPM from actions. Unclear if the publish won't work without the two new lines, but I will remove them as needed if there's a failure.